docs/docs/use-the-factory-orchestrator-api.md

```diff
@@ -0,0 +1,67 @@
+
+# Using the Factory Orchestrator client API
+
+The Factory Orchestrator service, FactoryOrchestratorService.exe, provides a robust API surface for clients to interact with test devices via .NET Standard code. You can use these APIs to author advanced task orchestration code to programmatically interact with the service outside of what the app provides.

```

Comment:
nit: Lets change to simply ".NET code" since with .NET 5, Net standard is deprecated.